_includes/modern-appendix.md

```diff
@@ -504,6 +533,20 @@ Sent to a client when joining the `<channel>` to inform them of the current [top
 
 Sent to a client to let them know who set the topic (`<nick>`) and when they set it (`<setat>` is a unix timestamp). Sent after [`RPL_TOPIC`](#rpltopic-332).
 
+### `RPL_WHOISACTUALLY (338)`
+
+      "<client> <nick> :is actually ...
+      "<client> <nick> <host|ip> :Is actually using host"
+      "<client> <nick> <username>@<hostname> <ip> :Is actually using host"
+
+Sent as a reply to the [`WHOIS`](#whois-message) command, this numeric shows details about the client with the nickname `<nick>`.
+
+`<username>` represents the name set by the [`USER`](#user-message) command (though `<username>` may be set by the server in other ways).
+
+`<host>` and `<ip>` represent the real host and IP address the client is connecting from. `<host>` CANNOT start with a colon `(':', 0x3A)` as this would get parsed as a trailing parameter – IPv6 addresses such as `"::1"` are prefixed with a zero `('0', 0x30)` to ensure this.
```

Comment:
The 0 prefix is actually a partial expansion of the IPv6 address and it might be a good idea to mention this.